storage/api/Storage.Samples/MakeBucketPublic.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_set_bucket_public_iam]
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+using System;
+using System.Collections.Generic;
+
+public class MakeBucketPublicSample
+{
+    public string MakeBucketPublic(
+        string bucketName = "your-unique-bucket-name")
+    {
+        var storage = StorageClient.Create();
+        var bucket = storage.GetBucket(bucketName); 
+
+        Policy originalPolicy = storage.GetBucketIamPolicy(bucketName);
+        var binding = originalPolicy.Bindings;
+
+        foreach(var bind in binding)
```

Comment:
I'm not certain this is what we should be doing here. This is not equivalent to what the Java sample does.
Take a look at the Java sample [here](https://github.com/googleapis/google-cloud-java/blob/7e04654a606e7c27db01bbad82cb0f38351cc490/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/MakeBucketPublic.java#L18-L44)

Reply:
Changed as discussed